arena.js

```diff
@@ -0,0 +1,42 @@
+class Arena{
+  constructor(arenaName) {
+    let arenaArray = arenaName.split('');
+      arenaArray[0] = arenaArray[0].toUpperCase();
+    let aName = arenaArray.join("");
+    this.name = aName;
+    this.gladiators = [];
+  }
+  addGladiator(name){
+    //Cannot add more than 2 gladiators
+    if (this.gladiators.length < 2){
+      this.gladiators.push(name);
+    } else {
+      console.error("Halt, You cannot add more than 2 gladiators");
+    }
+  }
+  fight(){
+    let glad1weap = this.gladiators[0].weapon
+    let glad2weap = this.gladiators[1].weapon
+    if ((glad1weap === "Spear" && glad2weap === "Club") ||
+       (glad1weap === "Trident" && glad2weap === "Spear") ||
+       (glad1weap === "Club" && glad2weap === "Trident")){
+      console.log(`${this.gladiators[0].name} Wins!`);
```

Comment:
How might you abstract this so you do not have such a big if statement? Is there any pattern between weapons...